package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "@kong-ui-public/i18n": "^0.8.0",
-    "@kong/kongponents": "^8.122.4",
+    "@kong/kongponents": "^8.122.6",
```

Comment:
**Issue**: With https://github.com/Kong/kongponents/pull/1641, this update is broken in kuma-gui because the `KCodeBlock` component is now missing an import of `KButton`. This breaks here because we don’t globally register components.